CITATION

```diff
@@ -1,4 +1,27 @@
-To cite the Common Workflow Language standard in a publication, please use:
+To cite the Common Workflow Language  and the CWL project in a scholary work, please use:
```

Comment:
Extra space but I believe this is fixed by Jekyll/markdown parsers/etc (`Language  and`)